client/src/reducers/getMealInfoReducer.js

```diff
@@ -0,0 +1,17 @@
+import { GET_MEAL_INFO_PENDING, GET_MEAL_INFO_FULFILLED, GET_MEAL_INFO_REJECTED} from '../actions/types';
+
+const initialState = {
+};
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case GET_MEAL_INFO_FULFILLED:
+      return {mealData:action.payload.data, status:"Fulfilled"};
```

Comment:
i think this is more self prefrence but i would have made an object of state like this -> 
```js 
{
isRejected: Boolean value,
isFulfilled: Boolean value,
isPending: Boolean value,
error: some error value,
data: some more values
}
```

but this wouldn't be the **entire state** this object would be in a certain key of the state.

Reply:
ohh it's actually in the docs this object i sound very shushpicious now